packages/genesys-spark-components/src/components/stable/gux-form-field/helper-components/gux-form-field-label-tooltip/gux-form-field-label-tooltip.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Element, h, JSX, Prop } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxFormFieldTooltipVariant } from './gux-form-field-label-tooltip.types';
+import { randomHTMLId } from '@utils/dom/random-html-id';
+
+@Component({
+  styleUrl: 'gux-form-field-label-tooltip.scss',
+  tag: 'gux-form-field-label-tooltip'
+})
+export class GuxFormFieldTooltip {
+  private tooltipId: string = randomHTMLId('gux-form-field-label-tooltip');
+
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  variant: GuxFormFieldTooltipVariant = 'info';
+
+  @Prop()
+  screenreaderText: string;
+
+  componentWillLoad(): void {
+    trackComponent(this.root);
+  }
+
+  render(): JSX.Element {
+    return (
+      <button
+        id={this.tooltipId}
+        type="button"
+        class="gux-form-field-label-tooltip"
```

Comment:
I chose this approach based on examples in two other design systems:
Spectrum - https://react-spectrum.adobe.com/react-aria/Tooltip.html
IBM - https://carbondesignsystem.com/components/tooltip/code/ 

This follows the pattern in other components like gux-datepicker where a button wraps a gux-icon.